templates/zerver/emails/missed_message.subject.txt

```diff
@@ -1,8 +1,8 @@
 {% if show_message_content %}
     {% if group_pm %} Group PMs with {{ huddle_display_name }}
     {% elif mention %} {{ sender_str }} mentioned you
-    {% elif private_message %} {{ sender_str }} sent you a message
-    {% elif stream_email_notify %} New messages in {{ messages[0].header.plain }}
+    {% elif private_message %} PMs with {{ sender_str }}
```

Comment:
We should fix the commit message typo: Improvise -> Improve (on both the commits that have that).